lib/Doctrine/Common/Collections/AbstractLazyCollection.php

```diff
@@ -157,6 +157,16 @@ public function toArray()
         return $this->collection->toArray();
     }
 
+    /**
+     * @return list<T>
+     */
+    public function toList(): array
+    {
+        $this->initialize();
+
+        return $this->collection->toList();
```

Comment:
We cannot be certain that the wrapped collection already implements the new method.